pom.xml

```diff
@@ -142,6 +142,7 @@
                             <goal>compile</goal>
                         </goals>
                         <configuration>
+							<extraJvmArgs>-Xmx1G</extraJvmArgs>
```

Comment:
Yes, at least for me with Maven + Eclipse + V8 I needed it, and found it documented in the V8 migration guide:

https://vaadin.com/vaadin-fw8-documentation-portlet/framework/migration/migrating-to-vaadin8.html

Perhaps my Eclipse setup uses a different, lower default configuration than however you ran it?